ui/privacy_page.go

```diff
@@ -0,0 +1,232 @@
+package ui
+
+import (
+	"gioui.org/layout"
+	"gioui.org/widget"
+	"gioui.org/widget/material"
+
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const PagePrivacy = "Privacy"
+
+type privacyPage struct {
+	theme         *decredmaterial.Theme
+	pageContainer layout.List
+	backButton    decredmaterial.IconButton
+	toggleMixer   *widget.Bool
+	infoBtn       decredmaterial.IconButton
+	line          *decredmaterial.Line
+}
+
+func (win *Window) PrivacyPage(common pageCommon) layout.Widget {
+	pg := &privacyPage{
+		theme:         common.theme,
+		pageContainer: layout.List{Axis: layout.Vertical},
+		backButton:    common.theme.PlainIconButton(new(widget.Clickable), common.icons.navigationArrowBack),
+		toggleMixer:   new(widget.Bool),
+		line:          common.theme.Line(),
+	}
+	pg.backButton.Color = common.theme.Color.Text
+	pg.backButton.Inset = layout.UniformInset(values.MarginPadding0)
+	pg.infoBtn = common.theme.IconButton(new(widget.Clickable), common.icons.actionInfo)
+	pg.infoBtn.Color = common.theme.Color.Gray
+	pg.infoBtn.Background = common.theme.Color.Surface
+	pg.infoBtn.Inset = layout.UniformInset(values.MarginPadding0)
+	pg.line.Color = common.theme.Color.Background
+	pg.line.Height = 1
+
+	return func(gtx C) D {
+		pg.Handler(common)
+		return pg.Layout(gtx, common)
+	}
+}
+
+func (pg *privacyPage) Layout(gtx layout.Context, common pageCommon) layout.Dimensions {
+	widgets := []func(gtx C) D{
+		func(gtx C) D {
+			return pg.header(gtx, &common)
+		},
+		pg.gutter,
+		func(gtx C) D {
+			return pg.mixerInfoLayout(gtx, &common)
+		},
+		pg.gutter,
+		func(gtx C) D {
+			return pg.mixerSettingsLayout(gtx, &common)
+		},
+		pg.gutter,
+		func(gtx C) D {
+			return pg.dangerZoneLayout(gtx, &common)
+		},
+	}
+
+	return common.Layout(gtx, func(gtx C) D {
+		return pg.pageContainer.Layout(gtx, len(widgets), func(gtx C, i int) D {
+			return widgets[i](gtx)
+		})
+	})
+}
+
+func (pg *privacyPage) header(gtx layout.Context, c *pageCommon) layout.Dimensions {
+	return c.theme.Card().Layout(gtx, func(gtx C) D {
+		gtx.Constraints.Min.X = gtx.Constraints.Max.X
+		return layout.UniformInset(values.MarginPadding10).Layout(gtx, func(gtx C) D {
+			return layout.Flex{}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.W.Layout(gtx, func(gtx C) D {
+						return layout.Inset{Right: values.MarginPadding20}.Layout(gtx, func(gtx C) D {
+							return pg.backButton.Layout(gtx)
+						})
+					})
+				}),
+				layout.Rigid(func(gtx C) D {
+					txt := pg.theme.H6("Privacy")
+					return txt.Layout(gtx)
+				}),
+				layout.Flexed(1, func(gtx C) D {
+					return layout.E.Layout(gtx, func(gtx C) D {
+						return pg.infoBtn.Layout(gtx)
+					})
+				}),
+			)
+		})
+	})
+}
```

Comment:
This layout is also used by the settings page. You can create a subpage layout with this header layout and wrap the privacy page in the subpage layout. There's already a subpage layout implemented in pages.go used by the verify message and sign message pages. You can create another one named "subpageSplitLayout"